### ISSUE TITLE
Can't get data from archive for short candece

### ISSUE THREAD

**Issue by AIofHuman** [KeplerGO/lightkurve#270]
#### Problem description

Good day! Thank a lot for your library! 
I try to get short cadence data from archive. But get error then I point month. I try to these code: month = [1], month = [1,2,3], month = 'all'. But always get error. Please help!
#### Example

```python
import lightkurve
from lightkurve import KeplerLightCurveFile
quartal = 7
KeplerID = 9146018
lcf1 = KeplerLightCurveFile.from_archive(KeplerID, quarter=quartal,cadence='short',month=[1])

#### Expected behavior
<!-- Describe the behavior you expected and how it differs from the behavior observed in the example. -->

#### Environment:
- platform - Windows 10:
- lightkurve version - 1.0b17
- installation method - pip:

**Comment by christinahedges** [comment]
Hi @AIofHuman! I can't seem to reproduce the error on my end, could you send a copy of the error message that you got?

Thanks!
C

**Comment by AIofHuman** [comment]
Hi Christina! Thanks a lot for you quick answer!
In attach, file copy of notebook with error message. Also I add fits file  KIC 9146018 for first month of 7 quarter.
Best regards,
Boris.

[attach.zip](https://github.com/KeplerGO/lightkurve/files/2435070/attach.zip)

**Comment by christinahedges** [comment]
Hi Boris,

Can you send a screenshot of the error message? Alternately you can upload your notebook [here](https://gist.github.com/) on gist so that we can see it as a webpage.

C

**Comment by AIofHuman** [comment]
Screenshot
![image](https://user-images.githubusercontent.com/41274676/46309222-672c0480-c5c4-11e8-82f7-27642a382428.png)

**Comment by christinahedges** [comment]
Hi Boris,

Thanks for the screenshot. This error isn't reproducible on our end, could you run the following and tell us what version number of lightkurve you are using?

``` python
import lightkurve
print(lightkurve.__version__)
```

**Comment by AIofHuman** [comment]
![image](https://user-images.githubusercontent.com/41274676/46309564-562fc300-c5c5-11e8-80ad-a3715f03dea2.png)

**Comment by christinahedges** [comment]
Hi Boris,

I'm running the same version and can't reproduce the error. 

![image](https://user-images.githubusercontent.com/14965634/46313061-bac63000-c57b-11e8-88c3-bbe1f783d638.png)

Sometimes MAST goes down and we can't download or query for files. Can you try to execute the same code again and verify you have the same error?

**Comment by AIofHuman** [comment]
Hi, Cristina! I try today - same result - error.  I think the problem in my low speed Internet connection. When I run same command for LC files - work fine, for SC files - error. One LC file is 400 kb, one SC file is 4000 kb.
On this week I will to another place with speed Internet connection and will check my suggetion.I will write about result.
Thanks.

**Comment by christinahedges** [comment]
@barentsen , do you think this could be timing out on a slow internet connection?

**Comment by AIofHuman** [comment]
Yes, I think so.

**Comment by AIofHuman** [comment]
Hi, Christina!
Today I tried to run my script via speed Internet connection, but it is failt (look at screen). I think that something happened at Mast. I will try later.
Best regards.
Boris.
![image](https://user-images.githubusercontent.com/41274676/46585760-8bcd2400-ca7d-11e8-9b22-d55bfa1d7dad.png)

**Comment by christinahedges** [comment]
This does look like a MAST problem. Please keep trying at let me know if you still can't connect.

**Comment by barentsen** [comment]
I activated the `from_archive` unit tests in Windows (AppVeyor) over in PR #280 and they seem to fail with an error message similar to that reported originally in this issue.  Travis (Linux) passes the same tests, so it looks like we have a Windows-specific bug in either `lightkurve` or `astroquery` that explains this problem.

**Comment by barentsen** [comment]
Good news: I believe to have resolved this bug!

After mitigating a large number of warnings which were being triggered during the unit tests (#338), I noticed the following warning which is unique to Appveyor/Windows (win32 platform):
```
lightkurve/tests/test_search.py::test_month
  C:\Miniconda3\envs\test-environment\lib\site-packages\astropy\io\ascii\fastbasic.py:120: AstropyWarning: OverflowError converting to IntType in column StartTime, reverting to String.
```

This indicated that `StartTime` was not being read successfully from `short_cadence_month_lookup.csv` on Windows, which is likely caused by the use of 32-bit instead of 64-bit `int` types on the `win32` platform.

I added, verified, and merged a workaround in #280.

I believe this issue can now be closed, but please re-open if the issue re-appears!  Thank you for reporting the bug @AIofHuman!
